src/firebolt/async_db/cursor.py

```diff
@@ -180,13 +177,7 @@ async def _do_execute(
         try:
             for query in queries:
                 start_time = time.time()
-                # Our CREATE EXTERNAL TABLE queries currently require credentials,
-                # so we will skip logging those queries.
-                # https://docs.firebolt.io/sql-reference/commands/create-external-table.html
-                if isinstance(query, SetParameter) or not re.search(
-                    "aws_key_id|credentials", query, flags=re.IGNORECASE
-                ):
-                    logger.debug(f"Running query: {query}")
+                self._log_query(query)
```

Comment:
I think `_log_query` is defined as static so calling it with `self` might break things?